### ISSUE TITLE
Calling GpC methylation with nanopolish

### ISSUE THREAD

**Issue by navtronius** [jts/nanopolish#853]
Is it possible to call GpC methylation using nanopolish instead of CpG methylation? (for an assay such as NOME-seq)

**Comment by jts** [comment]
Hi,

We have a combined CpG/GpC caller in the `cpggpc_new_train` branch. You can invoke combined CpG/GpC calling with `nanopolish call-methylation -q cpggpc`.

Jared

**Comment by kir1to455** [comment]
Hi,
When I used nanopolish to repeat the Nanome experiment, I found that my correlation was lower than Nanome to compare with WGBS.
Here is my nanopolish code:
/nanopolish-cpggpc_new_train/nanopolish call-methylation -q cpggpc -t 48 -r run.fastq.gz -b run.bam -g ${ref}

Also, I want to know the parameter, the difference between CpG and GPC and cpggpc.
In this example, I used cpggpc. Will call CpG alone be more accurate than call cpggpc together？

Thanks you for advance.
